bin/spark-evaluate.js

```diff
@@ -24,6 +24,8 @@ Sentry.init({
 
 assert(WALLET_SEED, 'WALLET_SEED required')
 
+await import('./migrate.js')
```

Comment:
I don't want to keep a long-running database connection since we write to the database only once per round (once per hour).

By importing the bin script, I don't have to write the code to open & close the PG connection.

Reply:
I agree with the connection thinking, what about turning this into a lib then though?

Reply:
Done in e3d3762